tests/app/main/views/test_services.py

```diff
@@ -253,7 +260,7 @@ def test_service_status_update_widgets_not_visible_when_not_permitted(self, data
 
 class TestServiceEdit(LoggedInApplicationTest):
     @mock.patch('app.main.views.services.data_api_client')
-    def test_edit_dos_service_title(self, data_api_client):
+    def test_edit_dos1_service_title(self, data_api_client):
```

Comment:
`dos1` isn't a thing - only sequels get numbered!! 😛

Reply:
And done